### ISSUE TITLE
[CI] TimeSeriesLifecycleActionsIT.testHistoryIsWrittenWithFailure failing to assert step

### ISSUE THREAD

**Issue by mark-vieira** [elastic/elasticsearch#52853]
There's an existing issue for this same test (https://github.com/elastic/elasticsearch/issues/50353) but that is failing on a different assertion so it's not clear if this is the same problem. If these both have the same root cause feel free to close this as a duplicate.

```
org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT > testHistoryIsWrittenWithFailure FAILED
    java.lang.AssertionError: 
    Expected: "ERROR"
         but: was "check-rollover-ready"
        at __randomizedtesting.SeedInfo.seed([25743638EB0838F4:A51AEC54A0F27B30]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.lambda$testHistoryIsWrittenWithFailure$104(TimeSeriesLifecycleActionsIT.java:1433)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:881)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:854)
        at org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.testHistoryIsWrittenWithFailure(TimeSeriesLifecycleActionsIT.java:1433)
REPRODUCE WITH: ./gradlew ':x-pack:plugin:ilm:qa:multi-node:integTestRunner' --tests "org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.testHistoryIsWrittenWithFailure" -Dtests.seed=25743638EB0838F4 -Dtests.security.manager=true -Dtests.locale=es-DO -Dtests.timezone=Asia/Aden -Dcompiler.java=13
```

https://gradle-enterprise.elastic.co/s/xmnuuygmxv4ky/tests/ao2nxxbijves6-iqisezgxfz7vq
https://gradle-enterprise.elastic.co/s/6lb3nya5vvutu/tests/ao2nxxbijves6-iqisezgxfz7vq
https://gradle-enterprise.elastic.co/s/rhh26e3fddsvq/tests/ao2nxxbijves6-iqisezgxfz7vq
https://gradle-enterprise.elastic.co/s/n5fq6qztklpb4/tests/ao2nxxbijves6-iqisezgxfz7vq

**Comment by jbaiera** [comment]
We've had a couple more failures of this specific test in the last couple of days:

https://gradle-enterprise.elastic.co/s/qhraswyh3qk6g
https://gradle-enterprise.elastic.co/s/cfyqrwx5isfjw

**Comment by mark-vieira** [comment]
@jbaiera Looks like the fix in #53039 didn't completely eliminate this. We should remute.
